examples/servers_detail_export_xlsx.py

```diff
@@ -46,7 +46,7 @@
         COMPUTER.write(ROW + 1, COL + 2, GROUPE_NAME)
 
     COMPUTER.write(ROW, COL + 3, "Status")
-    COMPUTER.write(ROW + 1, COL + 3, server.status["comment"])
+    COMPUTER.write(ROW + 1, COL + 3, server.status)
```

Comment:
no longuer the comment field ? what is the value ?

Reply:
can you give an example value* ?

Reply:
We do not use a comment field anymore

```
    "status": "server_vulnerable",
```